src/services/ffmpegService.ts

```diff
@@ -56,6 +56,38 @@ class FFmpegService {
             }
         }
     }
+
+    async convertLivePhotoToMP4(file: Uint8Array): Promise<Uint8Array> {
+        if (!this.ffmpeg) {
+            await this.init();
+        }
+        if (this.isLoading) {
+            await this.isLoading;
+        }
+
+        try {
+            this.ffmpeg.FS('writeFile', 'input.mov', file);
+            console.log('starting encoding', new Date().toLocaleTimeString());
```

Comment:
we should avoid logging things to the console. It's ok during testing but don't forget to remove them before deployment